tools.go

```diff
@@ -0,0 +1,9 @@
+// +build tools
```

Comment:
Can we add a comment in this file as to why it's required and what we hope it achieves?  Pretty confusing stuff here.

Reply:
Done